src/main/java/address/sync/SyncManager.java

```diff
@@ -28,18 +29,16 @@
     private RemoteManager remoteManager;
 
     public SyncManager() {
-        activeAddressBook = Optional.empty();
-        scheduler = Executors.newScheduledThreadPool(1);
-        requestExecutor = Executors.newCachedThreadPool();
-        EventManager.getInstance().registerHandler(this);
```

Comment:
I think this shouldn't be removed? Needed for `SyncManager` to handle certain events.